### PR TITLE
ddl: add description tidb ddl v2

### PR DESCRIPTION
**Pull request by GMHDBJD**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
- add tidb_ddl_version variable
- add description about tidb ddl v2

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: ddl-v2.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+1. 创建 `CREATE TABLE` Job
+
+    此步骤和 V1 版本实现无区别，通过解析 `CREATE TABLE` 语句生成相应的 DDL Job。
+
+2. 执行 `CREATE TABLE` Job
+
+    与 V1 版本将 Job 插入 `mysql.tidb_ddl_job` 表中不同，V2 版本由接收该 `CREATE TABLE` 语句的 TiDB 节点直接执行建表语句，将表结构持久化到 TiKV 中。同时，将 `CREATE TABLE` Job 标记为已完成，插入到 `mysql.tidb_ddl_history` 表中。
+
+3. 同步表信息
```
Comment (hfxsd) [inline ddl-v2.md:56]:
```suggestion
1. 创建 `CREATE TABLE` Job。

    此步骤和 V1 版本实现相同，通过解析 `CREATE TABLE` 语句生成相应的 DDL Job。

2. 执行 `CREATE TABLE` Job。

    与 V1 版本将 Job 插入 `mysql.tidb_ddl_job` 表中不同，V2 版本由接收该 `CREATE TABLE` 语句的 TiDB 节点直接执行建表语句，将表结构持久化到 TiKV 中。同时，将 `CREATE TABLE` Job 标记为已完成，插入到 `mysql.tidb_ddl_history` 表中。

3. 同步表信息。
```

File: system-variables.md [hunk 2]
```diff
@@ -1484,6 +1484,20 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
...
+> **警告：**
+>
+> 该变量目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
+- 默认值：`1`
+- 取值范围：`1` 或 `2`
+- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。开启后，将使用新版本的实现执行 DDL 语句。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。
```
Comment (hfxsd) [inline system-variables.md:1498]:
```suggestion
- 这个变量用于控制是否开启 [TiDB DDL V2](/ddl-v2.md)。将该变量的值设置为 `2` 可以开启该功能，设置为 `1` 可以关闭该功能。默认值为 `1`。开启后，将使用 TiDB DDL V2 执行 DDL 语句。
```

**Review by hfxsd — APPROVED** [review]

File: ddl-v2.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+要关闭该功能，将该变量的值设置为 `1`：
+
+```sql
+SET GLOBAL tidb_ddl_version = 1;
+```
+
+## 实现原理
+
+TiDB DDL V2 加速建表的详细的实现原理如下：
```
Comment (hfxsd) [inline ddl-v2.md:46]:
```suggestion
TiDB DDL V2 加速建表的详细的实现步骤如下：
```

File: ddl-v2.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+从 v7.6.0 开始，TiDB DDL 新版本 V2 实现支持加速建表，可提升大批量建表的速度。
+
+> **警告：**
+>
+> TiDB DDL V2 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+## 与 TiDB 工具的兼容性
+
+- [TiCDC](/ticdc/ticdc-overview.md) 暂不支持 TiDB DDL V2 加速建表。
```
Comment (hfxsd) [inline ddl-v2.md:16]:
```suggestion
- [TiCDC](/ticdc/ticdc-overview.md) 暂不支持同步通过 TiDB DDL V2 创建的表。
```

File: ddl-v2.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+>
+> TiDB DDL V2 目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+## 与 TiDB 工具的兼容性
+
+- [TiCDC](/ticdc/ticdc-overview.md) 暂不支持同步通过 TiDB DDL V2 创建的表。
+
+## 限制
+
+TiDB DDL V2 目前仅使用于 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md) 语句，且该建表语句不带任何外键约束。
```
Comment (hfxsd) [inline ddl-v2.md:24]:
```suggestion
TiDB DDL V2 目前仅适用于 [`CREATE TABLE`](/sql-statements/sql-statement-create-table.md) 语句，且该建表语句不带任何外键约束。
```

File: ddl-v2.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+## 使用方法
+
+可以通过设置系统变量 [`tidb_ddl_version`](/system-variables.md#tidb_ddl_version-从-v760-版本开始引入) 的值来开启或关闭 TiDB DDL V2 功能。
```
Comment (hfxsd) [inline ddl-v2.md:28]:
```suggestion
你可以通过设置系统变量 [`tidb_ddl_version`](/system-variables.md#tidb_ddl_version-从-v760-版本开始引入) 的值来开启或关闭 TiDB DDL V2 功能。
```

File: ddl-v2.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+要关闭该功能，将该变量的值设置为 `1`：
+
+```sql
+SET GLOBAL tidb_ddl_version = 1;
+```
+
+## 实现原理
+
+TiDB DDL V2 加速建表的详细的实现步骤如下：
```
Comment (hfxsd) [inline ddl-v2.md:44]:
```suggestion
TiDB DDL V2 加速建表的实现步骤如下：
```

File: ddl-v2.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
+---
+title: TiDB DDL V2 加速建表
+summary: 介绍 TiDB DDL V2 加速建表中的概念、原理、实现和影响。
+---
+
+# TiDB DDL V2 加速建表
+
+从 v7.6.0 开始，TiDB DDL 新版本 V2 实现支持加速建表，可提升大批量建表的速度。
+
+在 TiDB 中，对元数据对象的更改采用的是在线异步变更算法。所有的 DDL Job 会提交到 `mysql.tidb_ddl_job` 表里，由 owner 节点拉取 DDL Job，执行完 online DDL 算法中的各个阶段后，将该 DDL Job 标记为已完成，移入 `mysql.tidb_ddl_history` 表中。因此 DDL 只能在 owner 节点执行，无法线性拓展。
```
[the remark this answers is not in the document]
Comment (GMHDBJD) [reply ddl-v2.md:10]: 是的

Comment (hfxsd) [inline ddl-v2.md:10]:
```suggestion
在 TiDB 中，对元数据对象的更改采用的是  online DDL 算法（即在线异步变更算法）。所有的 DDL Job 会提交到 `mysql.tidb_ddl_job` 表里，由 owner 节点拉取 DDL Job，执行完 online DDL 算法中的各个阶段后，将该 DDL Job 标记为已完成，移入 `mysql.tidb_ddl_history` 表中。因此 DDL 只能在 owner 节点执行，无法线性拓展。
```

**Review by ran-huang — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Pull request merged**